timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -316,10 +316,25 @@ export default {
     },
     verboseAnalyzerOutput: function () {
       if (this.checkAnalyzerOutput) {
-        return JSON.parse(this.timeline.verdict)
+        // this can return null
+        const parsed = JSON.parse(this.timeline.verdict)
+      // normalize null to undefined
+        return parsed == null ? undefined : parsed;
       }
       return undefined
```

Comment:
During my tests I sometimes ran into an error, where this `verboseAnalyerOutput` function returned undefined even shortly after being assigned. Since it is not reproducible in a reliable way, my hypothesis is: There is a small chance for this function to return undefined before the analyzer output is parsed properly. 
I wonder if this could be solved with a small retry mechanism?

However, this is not blocking. If you have any idea to fix this, feel free to add it. If not, it is also good and can be fixed later on since it is not blocking.

Error message I got when triggering multiple analyzers for multiple timelines and having the results open on the left panel (including the detailed results) is below. I think this is triggered by the tooltip in line 81. `<span>Result Priority: {{ verboseAnalyzerOutput.result_priority }}</span>`

```
[Vue warn]: Error in render: "TypeError: Cannot read properties of undefined (reading 'result_priority')"

found in

---> <TsAnalyzerResultTimeline> at src/components/LeftPanel/AnalyzerResultTimeline.vue
       <TsAnalyzerResult> at src/components/LeftPanel/AnalyzerResult.vue
         <VData>
           <VDataIterator>
             <TsAnalyzerResults> at src/components/LeftPanel/AnalyzerResults.vue
               <VTabItem>
                 <VTabsItems>
                   <VNavigationDrawer>
                     <Sketch> at src/views/Sketch.vue
                       <VMain>
                         <VApp>
                           <App> at src/App.vue
                             <Root>


TypeError: Cannot read properties of undefined (reading 'result_priority')
    at Proxy.render (AnalyzerResultTimeline.vue:196:1)
    at Vue._render (vue.runtime.esm.js:2684:1)
    at VueComponent.updateComponent (vue.runtime.esm.js:3875:1)
    at Watcher.get (vue.runtime.esm.js:3446:1)
    at Watcher.run (vue.runtime.esm.js:3522:1)
    at flushSchedulerQueue (vue.runtime.esm.js:4121:1)
    at Array.eval (vue.runtime.esm.js:3143:1)
    at flushCallbacks (vue.runtime.esm.js:3065:1)
```

Reply:
Yes, it seems to be undefined in some cases.

We do not manually need to retry. If we use 'computed properties' like
```
resultPriority: function() {
    return this.verboseAnalyzerOutput && this.verboseAnalyzerOutput.result_priority
},
```

The change detection should update everything automatically as soon as  `this.verboseAnalyzerOutput` is defined.

I now fixed more places where we access `this.verboseAnalyzerOutput` in non safe ways.